en-wordlist.txt

```diff
@@ -554,3 +554,18 @@ pre
 otsu
 rgb
 histologically
+ACL
+BLAS
+Bfloat
+Graviton
+MKLDNN
+OpenBLAS
+SIMD
+SVE
+THP
+addmm
+aten
```

Comment:
I think these two should get resolved after indentation is fixed. Also, can you please sort alphabetically and then save. In vim, you should be able to just use `:sort` .
```suggestion
```